engine/verification/requester/requester.go

```diff
@@ -116,10 +133,6 @@ func (e *Engine) Process(originID flow.Identifier, event interface{}) error {
 
 // Ready initializes the engine and returns a channel that is closed when the initialization is done.
 func (e *Engine) Ready() <-chan struct{} {
-	if e.handler == nil {
-		e.log.Fatal().Msg("could not start requester engine with missing chunk data pack handler")
-	}
-
```

Comment:
Reasoning similar to [this](https://github.com/onflow/flow-go/pull/728/files#r635486216)